### ISSUE TITLE
Button: When the button has either set isToggle to false or undefined

### ISSUE THREAD

**Issue by JBismonte** [infor-design/enterprise-ng#1145]
**Describe the bug**
This is an issue raised by VPAT.
buttons that are has isToggle set to undefined still has aria-pressed=false attributes on the dom.

**To Reproduce**
1. Go to http://localhost:4200/ids-enterprise-ng-demo/button

**Expected behavior**
when buttons are not toggleable, aria-pressed attributes should be removed.

**Version**
- ids-enterprise-ng: 10.7.0

**Screenshots**
![image](https://user-images.githubusercontent.com/29673378/136197888-0b0e53c9-536a-4399-915f-95d81f66869b.png)

**Platform**
 - Device (if applicable): NA
 - OS Version: Windows 10
 - Browser Name : Firefox
 - Browser Version 
**Additional context**
https://jira.lawson.com/browse/LMCLIENT-37696

**Comment by tmcconechy** [comment]
This is definitely an issue in the NG component. It should remove this there.

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
